.github/workflows/test.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       matrix:
         platform: [ubuntu-latest, macos-latest]
-        toolchain: [stable, 1.69.0]
+        toolchain: [stable, 1.72.1]
```

Comment:
```bash

error[E0659]: `opentelemetry` is ambiguous
  --> /home/user/.cargo/registry/src/index.crates.io-6f17d22bba15001f/near-o11y-0.19.0/src/lib.rs:6:9
   |
6  | pub use opentelemetry::OpenTelemetryLevel;
   |         ^^^^^^^^^^^^^ ambiguous name
   |
   = note: ambiguous because of multiple potential import sources
   = note: `opentelemetry` could refer to a crate passed with `--extern`
   = help: use `::opentelemetry` to refer to this crate unambiguously
note: `opentelemetry` could also refer to the module defined here
  --> /home/jerrick/.cargo/registry/src/index.crates.io-6f17d22bba15001f/near-o11y-0.19.0/src/lib.rs:22:1
   |
22 | mod opentelemetry;
   | ^^^^^^^^^^^^^^^^^^
   = help: use `crate::opentelemetry` to refer to this module unambiguously
                                                                                                                                                                                    
For more information about this error, try `rustc --explain E0659`.
error: could not compile `near-o11y` (lib) due to previous error
```

Reply:
buildkite script has to be corrected  https://buildkite.com/nearprotocol/near-sdk-rs/builds/5218#018d0c88-ae78-49e7-8789-577def4a19da